crypto-keys/src/main/java/com/palantir/crypto2/keys/serialization/SymmetricKeySerializerV3.java

```diff
@@ -52,9 +52,11 @@
 enum SymmetricKeySerializerV3 implements SymmetricKeySerializer {
     INSTANCE;
 
-    private static final String AES_CTR_NO_PADDING = "AES/CTR/NoPadding";
-    private static final int IV_SIZE = 16;
-
+    private static final String AES_GCM_NO_PADDING = "AES/GCM/NoPadding";
+    private static final int IV_SIZE = 12;
```

Comment:
Realize no-one is probably using this yet, but we're breaking compat via algorithm & IV size switch. should we embed in serialized format?